dir.targets

```diff
@@ -18,6 +18,18 @@
   <!-- permit a wrapping build system to contribute targets to this build -->
   <Import Condition="Exists('$(MSBuildThisFileDirectory)..\open.targets')" Project="$(MSBuildThisFileDirectory)..\open.targets" />
 
+  <PropertyGroup>
+    <!-- We don't use any of MSBuild's resolution logic for resolving the framework, so just set these two properties to any folder that exists to skip
```

Comment:
There is a small chance of TFS build issues when we touch these files.  So we either merge and monitor TFS rolling builds, or I can buddy-build this in TFS and report back in a couple hours.